README.md

```diff
@@ -149,56 +158,294 @@ This application will prompt users to fill in three or four games that they like
+Go to part 2 to set up variables for MYSQL
+
+Part 1: Note: pip may not run within conda at times - Go to part 2 to set up variables for MYSQL
+requirements.txt may not run as pip may not be installed within conda(path not set).
+
+Create your own virtual environment.
+
+    conda create --name video_game pip
+    
+or use 
+    
+    virtualenv video_game 
+
+Activate virtual environment.
+
+    conda activate video_game
+or use:
+    
+    source video_game
+    
+Note: If you are in a virtual or base environment run the following:
+
+    pip install -r requirements.txt
+
+Part2:
+
+Edit your mysql config file accordingly
+
+    vi .mysqlconfig
+or use 
+
+    nano .mysqlconfig
+
+* Set MYSQL_USER to the "master username" that you used to create the database server.
+* Set MYSQL_PASSWORD to the "master password" that you used to create the database server.
+* Set MYSQL_HOST to be the RDS instance endpoint from the console
+* Set MYSQL_PORT to be 3306
+* SET DATABASE_NAME as the name of your rds instance if required
+
+Set up using the following within .mysqlconfig(please remove the caret '<' '>' signs:
+
+    export MYSQL_USER=<YOUR_USERNAME>
+    export MYSQL_PASSWORD=<YOUR_PASSWORD>
+    export MYSQL_HOST=<PASTE_YOUR_HOST_HERE>
+    export MYSQL_PORT=<3306>
+    export DATABASE_NAME=msia423asg5718
+
+
+Set the environment variables in your ~/.bashrc
+
+    echo 'source .mysqlconfig' >> ~/.bashrc
+
+    source ~/.bashrc
+    
+PLEASE NOTE: VERIFY THAT YOU ARE ON THE NORTHWESTERN VPN BEFORE YOU CONTINUE ON.        
+
+* SET AWS_ACCESS_KEY with your aws access key.
+* SET AWS_SECRET_KEY with you aws secret key.
+
+Run the following commands and replace xxxx with keys etc.
+
+    nano ~/.bashrc
+Note you may find 'source .mysqlconfig' already, leave it untouched.
+
+    export AWS_ACCESS_KEY=xxxx
+    export AWS_SECRET_KEY=xxxxx
+    source .mysqlconfig
+Don't forget to source:
 
-#### Create the database with a single song 
-To create the database in the location configured in `config.py` with one initial song, run: 
+    source ~/.bashrc
+Create environments variables for docker:
 
-`python run.py create_db --engine_string=<engine_string> --artist=<ARTIST> --title=<TITLE> --album=<ALBUM>`
+    cd config
+Open config.env file
 
-By default, `python run.py create_db` creates a database at `sqlite:///data/tracks.db` with the initial song *Radar* by Britney spears. 
-#### Adding additional songs 
-To add an additional song:
+    nano config.env
+Set credentials - replace the xxx here too :
 
-`python run.py ingest --engine_string=<engine_string> --artist=<ARTIST> --title=<TITLE> --album=<ALBUM>`
+    AWS_SECRET_KEY=xxxxxxxxxxxxxxxxx
+    AWS_ACCESS_KEY=xxxxxxxxxx
 
-By default, `python run.py ingest` adds *Minor Cause* by Emancipator to the SQLite database located in `sqlite:///data/tracks.db`.
+
+### 4.Set configurations:
+Go to config.py in the config directory:
+
+    cd config/ 
```

Comment:
shell commands are a lot more eyecatching when they're enclosed like this `cd config`

Reply:
Sorry, but they are in the README.md, do have a look.